variables.tf

```diff
@@ -15,7 +15,8 @@ variable "name" {
 
 variable "description" {
   type        = string
-  description = "(Required) Namespace description"
+  description = "Namespace description"
+  default = ""
```

Comment:
why we provide a default empty value for description

Reply:
generated in locals if empty
description = var.description != "" ? var.description : "${var.name} - ${var.namespace_type} namespace"